Dockerfile

```
@@ -12,8 +12,8 @@ ENV LANG=en_US.UTF-8
 ENV LC_ALL=C.UTF-8
 
 # environment variables for dependent binary versions
-ENV SYFT_VERSION=v0.33.0
-ENV GRYPE_VERSION=v0.27.3
+ENV SYFT_VERSION=v0.57.0
+ENV GRYPE_VERSION=v0.50.2
```

This could be already bumped to 0.55
```suggestion
ENV GRYPE_VERSION=v0.55.0
```